packages/thirdweb/package.json

```diff
@@ -169,6 +169,7 @@
     "@emotion/react": "11.11.4",
     "@emotion/styled": "11.11.0",
     "@google/model-viewer": "2.1.1",
+    "@noble/ed25519": "1.7.1",
```

Comment:
I think based on this we can likely also build our own `privateKeyAccount` instead of taking it from viem, since I believe it is already built on top of this

Reply:
I replaced this package with @noble/curves which contains both the ed25519 signature we need for farcaster signers and the secp256k1 signature we'll need for private key accounts